docs/mara_lampert/getting_started_with_mambaforge_and_python/readme.md

```diff
@@ -0,0 +1,199 @@
+# Getting started with Mambaforge and Python 
+[Mara Lampert](../readme), January 26th, 2023
+
+## Introduction to Python and Mamba 
+This blog post explains what Python and [Mamba](https://mamba.readthedocs.io/en/latest/installation.html)/ Mambaforge, and how you can download and setup it on your computer. We will also go through some steps how to get started with Bio-image Analysis. 
+
+> **_Note:_** This is an update of a [previous Blogpost](https://biapol.github.io/blog/johannes_mueller/anaconda_getting_started/) written by Johannes. 
+
+
+**Why do we need Mamba to use Python?**
+
+__Python__ is a programming language which is easy to learn and very important in scientific data analysis nowadays. 
+
+__Mamba__ is a __package manager__ which can be used with Python. It is a software allowing to install other software. Read more about Mamba [here](https://focalplane.biologists.com/2022/12/08/managing-scientific-python-environments-using-conda-mamba-and-friends/). 
```

Comment:
Maybe a short word about what makes mamba different from other distributions (e.g. Anaconda)? I would say that mambaforge is lighter and faster, but other people may come to different conclusions about that..

Reply:
People can read that in the linked blog post